packages/core/src/parsers/ComponentParser.js

```diff
@@ -642,7 +656,8 @@ class ComponentParser {
       const parser = new htmlparser.Parser(handler);
       const fileExt = utils.getExt(file);
       if (utils.isMarkdownFileExt(fileExt)) {
-        const renderedContent = md.render(content);
+        let renderedContent = md.render(content);
+        renderedContent = ComponentParser.convertMdExtToHtmlExt(renderedContent);
```

Comment:
you want to modify `_parse` so `html` files get the auto conversion too

Reply:
Shifted the invocation into ```_parse``` function!